### PR TITLE
[GITHUB-10] Update to Latest Standards

### PR DESCRIPTION
**Pull request by dennisconrad**
* Bump to v2.0
* Migrate to Molecule
* Update minimum Ansible version to 2.2
* Support Bionic

### CODE REVIEW

**Review by rsgoodall — APPROVED** [review]

File: tasks/build.yml [hunk 1]
```diff
@@ -3,12 +3,12 @@
 - name: Assert vendor variable
   assert:
     that:
-      - java.vendor in [ 'oracle', 'openjdk' ]
+      - sansible_java_vendor in [ 'oracle', 'openjdk' ]
 
 - name: Build Oracle Java
   include: build/build-oracle.yml
-  when: java.vendor == 'oracle'
+  when: "sansible_java_vendor == 'oracle'"
```
Comment (lobsterdore) [inline tasks/build.yml]: I think when clauses should be left bare (ie. unquoted) I might be wrong on that though.

**Comment by lobsterdore** [comment]
Just one tiny comment, looks good otherwise.

File: tasks/build.yml (hunk 1 above)
Comment (dennisconrad) [reply tasks/build.yml]: Seen both and I think `"'foo' in bar"` requires them, but I'm not fighting this one :-)

**Comment by lobsterdore** [comment]
LGTM 👍

**Pull request merged**